.github/workflows/deploy_environment.yml

```diff
@@ -10,12 +10,17 @@ on:
       commit-identifier:
         description: SHA or tag to apply, (optional) when omitted, the head of the selected branch is used
         type: string
-        required: true
+        required: false
```

Comment:
I think this should stay required. Instead of defaulting to github.sha in line 78 below, you could have passed that in as the parameter value. I think this is probably what identity-infra and/or identity-userstore-infra are doing

Reply:
If the most common use case is going to be github.sha, then why make it required?